src/renderers/WebGLRenderer.js

```diff
@@ -2357,6 +2357,85 @@ class WebGLRenderer {
 
 		};
 
+		this.readRenderTargetPixelsAsync = async function ( renderTarget, x, y, width, height, buffer, activeCubeFaceIndex ) {
```

Comment:
Apart from documenting the new method the PR looks good!

Reply:
Added!